src/views/play/sardines.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <smoo-card
+    class="smoo-view-play-sardines"
+    :visible="$attrs.visible"
+    accordion="play"
+    key="sardines"
+    @show="$emit('show', $event)"
+    @hidden="$emit('hidden', $event)"
+  >
+    <template #header><smoo-role-icon type="sardine"/> <b>Sardines</b> game mode</template>
+    <p>
+      The <b>Sardines</b> game mode was developed by
+      <a-ext href="https://github.com/Amethyst-szs/SuperMarioOdysseyOnline/tree/sardines" icon="github">Amethyst-szs</a-ext>
+      in 2022 and was added to the official mod with version <code>v1.6.0</code>.
```

Comment:
This version number might need to be changed, if the first release with `Sardines & Freeze-Tag` is not `v1.6.0`.